pymars/tests/test_methods.py

```diff
@@ -0,0 +1,74 @@
+""" Tests the reduction methods implemented by pyMARS """
+
+import sys
+import os
+sys.path.append(os.path.dirname(os.path.abspath(__file__)) + "/..")
+
+import cantera as ct
+
+import drgep
+import pfa
+import sensitivity_analysis
+
+
+def testDRGEPSA():
+
+	# Original model
+	path_to_original = os.path.dirname(os.path.abspath(__file__)) + "/../../example_files/gri30.cti" 
```

Comment:
this is fairly fragile, and I think there's a better way using `pkg_resources.resource_ filename()`.

In this case, something along the lines of
```
    def relative_location(self, file):
        file_path = os.path.join("example_files", file)
        return pkg_resources.resource_filename(__name__, file_path)
```
might work

Reply:
Had trouble getting this to work, but I tried something else and I think it is a little cleaner now.  See latest commit.